### PR TITLE
Feature/oracle plugin

### PR DESCRIPTION
**Pull request by ibarbashov**

### CODE REVIEW

File: oracle-plugin/docs/Oracle-batchsink.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Oracle Batch Sink
+
+
+Description
+-----------
+Writes records to Oracle table. Each record will be written to a row in the table.
```
Comment (albertshau) [inline oracle-plugin/docs/Oracle-batchsink.md]: an Oracle

File: oracle-plugin/docs/Oracle-batchsink.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+Use Case
+--------
+This sink is used whenever you need to write to a Oracle table.
```
Comment (albertshau) [inline oracle-plugin/docs/Oracle-batchsink.md]: a Oracle -> an Oracle

File: oracle-plugin/docs/Oracle-batchsink.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Suppose you periodically build a recommendation model for products on your online store.
+The model is stored in a FileSet and you want to export the contents
+of the FileSet to a Oracle table where it can be served to your users.
+
+Column names would be autodetected from input schema.
```
Comment (albertshau) [inline oracle-plugin/docs/Oracle-batchsink.md]: don't need this sentence

File: oracle-plugin/docs/Oracle-batchsource.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+Description
+-----------
+Reads from Oracle table using a configurable SQL query.
+Outputs one record for each row returned by the query.
+
+
+Use Case
+--------
+The source is used whenever you need to read from a Oracle. For example, you may want
```
Comment (albertshau) [inline oracle-plugin/docs/Oracle-batchsource.md]: from an Oracle database

**Review by albertshau — COMMENTED** [review]
minor comments about docs

File: oracle-plugin/docs/Oracle-batchsource.md [hunk 5]
```diff
@@ -3,7 +3,7 @@
 
 Description
 -----------
-Reads from a Oracle using a configurable SQL query.
+Reads from Oracle table using a configurable SQL query.
```
Comment (albertshau) [inline oracle-plugin/docs/Oracle-batchsource.md]: from Oracle -> from an Oracle

File: oracle-plugin/docs/Oracle-postaction.md [hunk 6]
```diff
@@ -3,7 +3,7 @@
 
 Description
 -----------
-Runs a Oracle query at the end of the pipeline run.
+Runs Oracle query at the end of the pipeline run.
```
Comment (albertshau) [inline oracle-plugin/docs/Oracle-postaction.md]: Runs Oracle -> Runs an Oracle

Comment (ibarbashov) [reply oracle-plugin/docs/Oracle-postaction.md]: Thanks, Albert. Is there any rule of thumb that we may stick to. For example we use an/a when referencing something for the first time in text and use 'the' or no article in following sentences. 

Should we use articles in new paragraph when writing about the same thing? Also, why don't we use articles in @Description annotations?

Comment (albertshau) [reply oracle-plugin/docs/Oracle-postaction.md]: This is just a grammar thing where you don't say 'Runs query', but 'Runs a query'.

Comment (albertshau) [reply oracle-plugin/docs/Oracle-postaction.md]: This still needs to be fixed btw

File: oracle-plugin/docs/Oracle-batchsink.md (hunk 1 above)
Comment (ibarbashov) [reply oracle-plugin/docs/Oracle-batchsink.md]: done

File: oracle-plugin/docs/Oracle-batchsink.md (hunk 2 above)
Comment (ibarbashov) [reply oracle-plugin/docs/Oracle-batchsink.md]: done

File: oracle-plugin/docs/Oracle-batchsink.md (hunk 3 above)
Comment (ibarbashov) [reply oracle-plugin/docs/Oracle-batchsink.md]: done

File: oracle-plugin/docs/Oracle-batchsource.md (hunk 5 above)
Comment (ibarbashov) [reply oracle-plugin/docs/Oracle-batchsource.md]: done

File: oracle-plugin/docs/Oracle-postaction.md (hunk 6 above)
Comment (ibarbashov) [reply oracle-plugin/docs/Oracle-postaction.md]: done

**Comment by albertshau** [comment]
please resolve the conflicts in the pom, otherwise lgtm

**Pull request merged**